.gitignore

```diff
@@ -79,4 +79,14 @@ www/
 .tmp/
 
 # npm package-locks
-package-lock.json
\ No newline at end of file
+package-lock.json
+
+# Docusaurus dependencies
+lib/core/metadata.js
+lib/core/MetadataBlog.js
+
+website/translated_docs
+website/build/
+website/yarn.lock
+website/node_modules
+website/i18n/*
```

Comment:
Would be great if in a follow-up we can just ignore:

```
www/
lib/
```

Reply:
There are files additional files in here which are not auto-generated and contain dynamic code.  It doesn't really make much sense but it's structure this way.
`/docs` contains markdown, `/website/blog`, `/website/core`, `/website/pages`, `/website/static`, all are dynamic used to control some aspect of the site.  The site then generates static files under `/website/build` which ends up containing folders equivalent to the gh-pages branch when publishing.

Reply:
Won't fix, though I will investigate `lib/core/` etc, not sure what these are.